dist/main.js

```diff
@@ -0,0 +1,15 @@
+
+const manager = new dataManager()
+const renderer = new Renderer()
+
+const defaultImage = function(img){
+    img.src = 'nba.png'
+    $(img).css('height','254px')
+    $(img).css('width','350px')
+}
+
+const displayPlayers = function(){
+    manager.players()
+     setTimeout(function(){renderer.renderPlayer(manager.data)}, 500)
```

Comment:
this shouldn't be done with setTimeout, it should be done with a callback function or a `.then` function after invoking the `manager.players()` method